### DISCUSSION TITLE
No CSV files available in the folder

### DISCUSSION THREAD

**Discussion by chxyike** [rodralez/NaveGo#142] [Q&A] [answered]
When I run the examples, it gave the error message like:

Error using readtable (line 197)
Unable to open file './external-data/canada/pose_data.csv'.

Error in visual_structure (line 23)
visual_data_planetary_pose = readtable('./external-data/canada/pose_data.csv');

Error in navego_example_canada_data (line 107)
        visual_structure;

Any ideas?

**Comment by rodralez** [comment, marked as answer]
Hello,

Thank you for point out this problem. I am working to solve it ASAP.

**Reply by rodralez** [reply]
This problem was solved with PR #143.
